### PR TITLE
#605 - Verify publish in NpmITCase

### PR DESCRIPTION
**Pull request by genryxy**
Part of #605 
Verify that npm publish project without authentication is working.

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmITCase.java [hunk 1]
```diff
@@ -114,6 +117,41 @@ void npmInstall() throws Exception {
...
+        final boolean anonymous = true;
+        final String proj = "@hello/simple-npm-project";
+        final Key path = new Key.From("repos/my-npm");
+        this.init(anonymous);
+        new TestResource("npm/simple-npm-project")
+            .addFilesTo(this.storage, new Key.From(proj));
+        MatcherAssert.assertThat(
+            "Package was published",
+            this.exec("npm", "publish", proj, "--registry", this.url),
+            new StringContains("+ @hello/simple-npm-project@1.0.1")
```
Comment (olegmoz) [inline src/test/java/com/artipie/npm/NpmITCase.java:131]: @genryxy it looks like `proj` is part of the string, I'd suggest to use it

File: src/test/java/com/artipie/npm/NpmITCase.java [hunk 2]
```diff
@@ -114,6 +117,41 @@ void npmInstall() throws Exception {
...
+                .getJsonObject("1.0.1")
+                .getJsonObject("dist")
+                .getString("tarball"),
+            new IsEqual<>(String.format("/%s/-/%s-1.0.1.tgz", proj, proj))
+        );
+        MatcherAssert.assertThat(
+            "File should be in storage after publishing",
+            this.storage.exists(
+                new Key.From(path, String.format("%s/-/%s-1.0.1.tgz", proj, proj))
+            ).toCompletableFuture().join(),
```
Comment (olegmoz) [inline src/test/java/com/artipie/npm/NpmITCase.java:150]: @genryxy `.tgz` file path is constructed twice here, I'd suggest to extract into variable

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see two comments

**Comment by genryxy** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#717` is not assigned, can't get performer

**Pull request merged**